src/libcore/slice.rs

```diff
@@ -880,6 +957,15 @@ impl<'a, T> ops::IndexMut<ops::FullRange> for IterMut<'a, T> {
         make_slice!(T => &mut [T]: self.ptr, self.end)
     }
 }
+#[cfg(not(stage0))]
+#[unstable]
+impl<'a, T> ops::IndexMut<ops::RangeFull> for IterMut<'a, T> {
+    type Output = [T];
+    #[inline]
+    fn index_mut(&mut self, _index: &ops::RangeFull) -> &mut [T] {
+        make_slice!(T => &mut [T]: self.ptr, self.end)
+    }
+}
```

Comment:
Actually, could most of the duplication be done via the renaming import above?

It just ends up reducing merge conflicts is all :)